website/docs/validation/host-network-ports.md

```diff
@@ -16,7 +16,7 @@ metadata:
   name: k8spsphostnetworkingports
   annotations:
     metadata.gatekeeper.sh/title: "Host Networking Ports"
-    metadata.gatekeeper.sh/version: 1.1.1
+    metadata.gatekeeper.sh/version: 1.2.0
```

Comment:
As this is a Rego "fix", probably just bump the patch?

"Patch version bump required: Whenever there is a simple backward compatible change in the policy, e.g. Simple Rego fix, updating policy metadata"

```suggestion
    metadata.gatekeeper.sh/version: 1.1.2
```